compiler/rustc_hir_typeck/src/method/confirm.rs

```diff
@@ -503,7 +503,7 @@ impl<'a, 'tcx> ConfirmContext<'a, 'tcx> {
 
         debug!("method_predicates after subst = {:?}", method_predicates);
 
-        let sig = self.tcx.bound_fn_sig(def_id);
+        let sig = self.tcx.fn_sig(def_id);
 
         let sig = sig.subst(self.tcx, all_substs);
```

Comment:
I think with the shorter name for `bound_fn_sig` we can again write it as the following in one line
```suggestion
        let sig = self.tcx.fn_sig(def_id).subst(self.tcx, all_substs);
```